src/openforms/js/components/admin/form_design/PriceLogic.js

```diff
@@ -39,38 +40,51 @@ const PRICING_MODES = [
       defaultMessage: 'Use logic rules to determine the price',
     }),
   ],
+  [
+    'fromVariable',
```

Comment:
This should match the value in the backend (so `from_variable` rather than the camelCase).